WBooks/Book_Information/BookInformationViewModel.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  BookInformationViewModel.swift
+//  WBooks
+//
+//  Created by Carolina Arcos on 9/3/18.
+//  Copyright © 2018 Wolox. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import ReactiveSwift
+import Result
+
+class BookInformationViewModel {
+    
+    // MARK: - Properties
+    
+    private let book: Book
+    let image = MutableProperty(UIImage(named: "default_image"))
+    
+    var title: String {
+        return book.title
+    }
+    
+    var status: String {
+        return book.available ? "Avaliable" : "Not Avaliable"
```

Comment:
With the new Apple's interface, it's a good practice to name these things like: `isAvailable`. Which is clearer and more unambiguous than just `available`.